src/libs/molecules/card-promo/card-promo.ce.vue

```diff
@@ -8,6 +8,9 @@
     <div class="ma-card-promo__text-wrapper">
       <label>
         {{ data.title }}
+        {{ data.title }}
```

Comment:
sorry ris, ini dikit lagi kelewat. 
lupa lu hapus yah, thank you banget.

Reply:
ohiya lupa gw mas